src/store/routeAssets/actions.ts

```diff
@@ -60,6 +60,7 @@ const actions = defineActions({
           // console.log((row.meta.cursor / file.size) * 100);
           try {
             const amountInTokens = row.data[4] ? row.data[4].trim().toLowerCase() === 'true' : false;
```

Comment:
why boolean value is represented as string here? if it was from localStorage (or anywhere else) you can use JSON.parse. if not - can you please modify this type?